README.md

```diff
@@ -7,24 +7,49 @@ The PyFingerprint library allows to use ZhianTec ZFM-20, ZFM-60, ZFM-70 and ZFM-
 **Note:** The library is inspired by the C++ library from Adafruit Industries:
 <https://github.com/adafruit/Adafruit-Fingerprint-Sensor-Library>
 
-## Package building on Debian
+## Installation
+
+There are two ways of installing PyFingerprint: Installation of the stable or latest version. The stable version is distributed through the PM Code Works APT repository and is fully tested but does not contain the latest changes.
```

Comment:
Better change "latest version" to "development version" - because in my opinion stable and latest is too close in meaning.